test/drenv/kubectl.py

```diff
@@ -197,6 +197,8 @@ def gather(contexts, namespaces=None, directory=None):
         cmd.extend(("--namespaces", ",".join(namespaces)))
     if directory:
         cmd.extend(("--directory", directory))
+    if verbose:
+        cmd.append("--verbose")
     commands.run(*cmd)
```

Comment:
Currently we buffer the output from kubectl gather - we need to watch its stderr instead and log the returned lines.

We also want to get json logs from kubectl gather so we can log only the message.